outputs.tf

```
@@ -9,6 +9,11 @@ output "state_machine_arn" {
   value       = try(aws_sfn_state_machine.this[0].arn, "")
 }
 
+output "state_machine_name" {
+  description = "The Name of the Step Function"
+  value       = try(var.name, "")
```

```suggestion
  value       = try(aws_sfn_state_machine.this[0].name, "")
```

This is correct because we should not just return `var.name`. We need to make sure that we get value from the resource (mostly to respect the graph dependencies and be able to use in `depends_on`).